rewrite-java-test/src/test/java/org/openrewrite/java/ReorderMethodArgumentsTest.java

```diff
@@ -123,15 +108,14 @@ public void test() {
     @Test
     void reorderArgumentsWhereOneOfTheOriginalArgumentsIsVararg() {
         rewriteRun(
-          spec -> spec.recipe(new ReorderMethodArguments("a.A foo(..)",
-              new String[]{"s", "o", "n"}, null, null, null))
-            .cycles(1).expectedCyclesThatMakeChanges(1),
+          spec -> spec.recipe(new ReorderMethodArguments("a.A foo(String,Integer,..)",
```

Comment:
had to make the matcher a bit more specific, otherwise it was always matching no matter if we changed the method Type